intermediate-python-debugging/debugging-pytest.md

```diff
@@ -0,0 +1,40 @@
+# Debugging Tests With Pytest
+
+
+All the previous lessons on debugging apply to running tests, but VS Code has the ability to run individual tests from the editor and it allows you to run tests in the debugger.  This can help you get tests to pass and identify why they fail.
+
+## Learning Goals
+
+By the end of this lesson, we will be able to...
+
+- Run the debugger on individual tests
```

Comment:
As mentioned in others:

- I'd prefer this to be called "Goals"
- SWBAT statement hasn't been in the other lessons...
```suggestion
## Goals

We have practiced using a debugger tool while running Python programs. Our goal is to apply those same debugger skills when we are running unit tests.

This lesson is structured like a follow-along exercise to illustrate:

- Setting up Pytest for the VS Code debugger
- Running tests using the VS Code debugger

Besides setup, overall we should observe that our familiar tools of setting breakpoints, running, pausing, and debugging, are overall similar between tests and programs.
```